From: Antoine Beaupre <anarcat@debian.org>
Date: Wed, 14 Mar 2018 21:34:55 +0100
Subject: Disable version checks on startup

Origin: upstream, https://discuss.pixls.us/t/rapid-photo-downloader-0-9-9-is-released/6912
Forwarded: not-needed

Rapid-photo-downloader performs a check for new versions on startup
upstream. In Debian, users cannot perform this update themselves
through the mechanisms offered upstream: they must perform updates
through the Debian packaging system. This check will, when the
package trickle down in Debian, trigger needless warnings when
upstream performs new releases that are not backported to stable or
LTS releases.
---
 raphodo/constants.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/raphodo/constants.py b/raphodo/constants.py
index 8361d55..59e4c79 100644
--- a/raphodo/constants.py
+++ b/raphodo/constants.py
@@ -31,7 +31,7 @@ remote_versions_file = 'https://www.damonlynch.net/rapid/version.json'
 
 # If set to True, the ability to check for a new version will be removed
 # from the user interface and disabled in program logic.
-disable_version_check = False
+disable_version_check = True
 
 
 class CheckNewVersionDialogResult(IntEnum):
